docs/source/release/v6.6.0/Direct_Geometry/General/New_features/34748.rst

```diff
@@ -0,0 +1 @@
+Added `ILLLagrange`, which reduces data from IN1 - Lagrange at ILL.
```

Comment:
Could you add here a reference to the algorithm so that it connects to the documentation?